stac_mjx/compute_stac.py

```diff
@@ -51,7 +51,10 @@ def root_optimization(
     # rodent.xml, corresponding to the index of 'SpineL' keypoint.
     # For the mouse model this should be 3*5, corresponding 'Trunk'
     root_kp_idx = 3 * 18
+    # FLY_MODEL: 
+    # root_kp_idx = 0
     q0.at[:3].set(kp_data[frame, :][root_kp_idx : root_kp_idx + 3])
+    # q0.at[:3].set(jp.zeros(3))
```

Comment:
@elliottabe Is this line only relevant for tethered fly?

Reply:
What do you mean only relevant for the tethered fly? the root_kp_idx  will probably be different for most models so having it not be a hard coded value is fine. If you mean the jp.zeros(3) part that was just easier since the thorax (root) is at (0,0,0)